R/dictionaries.R

```diff
@@ -574,37 +592,48 @@ list2dictionary <- function(dict) {
 }
 
 
-# Import a Lexicoder dictionary
-# dict <- read_dict_lexicoder('/home/kohei/Documents/Dictionary/Lexicoder/LSDaug2015/LSD2015.lc3')
+#' Import a Lexicoder dictionary
+#' 
+#' @param path the path from which to read in the Lexicoder dictionary
+#' @keywords dictionary internal
+#' @examples
+#' \dontrun{
+#' # dict <- read_dict_lexicoder("/home/kohei/Documents/Dictionary/Lexicoder/LSDaug2015/LSD2015.lc3")
```

Comment:
How about we add these files to `extdata` or somewhere that they can be accessed if a user wants to try them? (And when running `devtools::run_examples()`? I will do the same for my local file references!

Reply:
I'll make this a separate PR.

Reply:
Moved to #1507.